### PR TITLE
styles adjustments

### PR DESCRIPTION
**Pull request by preludium**

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/components/AttendedProcessModal/AttendedProcessModal.tsx [hunk 1]
```diff
@@ -33,9 +33,8 @@ const AttendedProcessModal: React.FC<UserModalProps> = ({ open, setOpen, process
     const processForm = useMemo(() => {
         if (isJsonValid(process?.executionInfo)) {
             return JSON.parse(process.executionInfo);
-        } else {
-            return null;
         }
+        return null;
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/components/AttendedProcessModal/AttendedProcessModal.tsx:37]: nojs

File: runbotics/runbotics-orchestrator-ui/src/components/Table/Table.tsx [hunk 2]
```diff
@@ -118,7 +118,7 @@ const Table = <T extends object>({
             const rowKey = row.getRowProps().key;
             return (
                 <React.Fragment key={rowKey}>
-                    <DataTableRow isRowSelected={row.isSelected} isSubRoww={row.depth > 0}>
+                    <DataTableRow isClickable={!!onRowClick} isRowSelected={row.isSelected} isSubRoww={row.depth > 0}>
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/components/Table/Table.tsx]: I think casting with `Boolean(val)` is better for readabilty, what do u think?

File: runbotics/runbotics-orchestrator-ui/src/components/Table/Table.styles.ts [hunk 3]
```diff
@@ -23,32 +23,32 @@ export const DataTableWrapper = styled.div(({ theme }) => `
...
     }
 `;
 
-const rowStyles = css<{ isRowSelected?: boolean }>`
+const rowStyles = css<{ isRowSelected?: boolean, isClickable?: boolean }>`
     ${(p) => p.isRowSelected && `background-color: ${p.theme.palette.action.selected}`};
 
     &:hover {
-        cursor: pointer;
+        cursor: ${(p) => (p.isClickable ? 'pointer' : 'normal')};
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/components/Table/Table.styles.ts]: i think `props` or destructuring would be more consistent with the rest of the project

File: runbotics/runbotics-orchestrator-ui/src/components/ResizableDrawer/ResizableDrawer.tsx [hunk 4]
```diff
@@ -1,24 +1,23 @@
...
 
     const handleDragStart = () => {
         setDragmode(true);
     };
 
     const handleMouseMove = (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
-        setWidth(window.innerWidth - event.clientX);
+        setTimeout(() => {
+            setWidth(window.innerWidth - event.clientX);
+        }, 100);
```
Comment (FranekMarciniak) [inline runbotics/runbotics-orchestrator-ui/src/components/ResizableDrawer/ResizableDrawer.tsx]: why do we need timeout here?

**Review by FranekMarciniak — CHANGES_REQUESTED** [review]

File: runbotics/runbotics-orchestrator-ui/src/components/Table/Table.tsx (hunk 2 above)
Comment (preludium) [reply runbotics/runbotics-orchestrator-ui/src/components/Table/Table.tsx]: +1, can cause a mistakes

File: runbotics/runbotics-orchestrator-ui/src/components/ResizableDrawer/ResizableDrawer.tsx (hunk 4 above)
Comment (preludium) [reply runbotics/runbotics-orchestrator-ui/src/components/ResizableDrawer/ResizableDrawer.tsx]: resize dragger was lagging on my local machine then I found it helpful to reduce the problem but tbh it is not a problem on dev or prod so I will undo

**Review by FranekMarciniak — DISMISSED** [review]

**Pull request merged**